pkgs/by-name/gi/git-cola/package.nix

```
@@ -14,32 +14,32 @@
 
 python3Packages.buildPythonApplication rec {
   pname = "git-cola";
-  version = "4.15.0";
+  version = "4.16.0";
   pyproject = true;
 
   src = fetchFromGitHub {
     owner = "git-cola";
     repo = "git-cola";
     tag = "v${version}";
-    hash = "sha256-h3W7CsdJK1hid8Nmp1bvFwiHVS4UV/gziwtyZuxSxHY=";
+    hash = "sha256-gBqMwqmpu0+gMeffiFdwy/kBdCUQRpJr+3vzUkBCRSk=";
   };
 
-  buildInputs = lib.optionals stdenv.hostPlatform.isLinux [ qt5.qtwayland ];
+  buildInputs = lib.optionals stdenv.hostPlatform.isLinux [ qt6.qtwayland ];
```

```suggestion
  buildInputs = [
    qt6.qtbase
  ]
  ++ lib.optionals stdenv.hostPlatform.isLinux [ qt6.qtwayland ];
```
I think otherwise the wrapper fails on darwin